openedx_external_enrollments/tests/tests_backends.py

```diff
@@ -0,0 +1,13 @@
+"""This file contains all the necessary backend in a test scenario."""
+
+
+class ApiKeyHeaderPermissionIsAuthenticated:
+    """Test class for openedx.core.lib.api.permissions.ApiKeyHeaderPermissionIsAuthenticated"""
+
+
+class JwtAuthentication:
+    """Test class for edx_rest_framework_extensions.auth.jwt.authentication.JwtAuthentication"""
+
+
+def get_course_enrollment_backend():
+    """Test get_course_enrollment_backend method."""
```

Comment:
pass? @andrey-canon just to be explicit.

Reply:
Unnecessary pass statement, but it has not explanation on why we should avoid the pass statement... anyway let's keep it as it is. @andrey-canon